test/Microsoft.IdentityModel.Tokens.Tests/SignatureProviderTests.cs

```diff
@@ -621,14 +621,14 @@ public static TheoryData<SignatureProviderTheoryData> SymmetricVerifySingatureSi
                         ExpectedException = EE.ArgumentException("IDX10719:"),
                         RawBytes= new byte[32],
                         Signature = new byte[32],
-                        SigningSignatureProvider = new SymmetricSignatureProvider(KEY.SymmetricSecurityKey2_256, ALG.HmacSha384),
```

Comment:
>SigningSignatureProvider = new SymmetricSignatureProvider(KEY.SymmetricSecurityKey2_256, ALG.HmacSha384), [](http://example.com/codeflow?start=24&length=105)

So this call used to throw `ArgumentException` but now throws `InvalidOperationException`. Let's make sure to call this out in release notes.

I know Karel would say we should bump the major version for this. We want to at least make it explicit when we release though since we know some partners will consider this breaking.

Reply:
@TimHannMSFT it will still throw argument exception when the algorithm is not known.
A new exception will be thrown InvalidOperationException when the key size is not correct.
I will review this to see if we can throw an existing exception.

Reply:
Not sure I follow. When I run these tests they're not throwing any exceptions and the test doesn't seem to care (at least `SymmetricVerify1Tests` e.g. that the Theory data contains an `ExpectedException`.

That test has Verify return `false` and since no exception is thrown it's never checked. This is what's happening in dev too though so I guess it's no worse than what's checked it. This seems like confusing test behavior to me and...SEEMS to allow for a case where someone sets an expected exception in the theory data and tests silently succeed which seems problematic. In any case, given that part isn't a behavior change I don't want to hold this up.